qiwis.py

```diff
@@ -194,39 +198,48 @@ def addFrame(self, name: str, frame: QWidget, info: AppInfo):
             frame: A frame to show.
             info: An AppInfo object describing the app.
         """
-        dockWidget = QDockWidget(name, self.mainWindow)
-        dockWidget.setWidget(frame)
-        area = {
-            "left": Qt.LeftDockWidgetArea,
-            "right": Qt.RightDockWidgetArea,
-            "top": Qt.TopDockWidgetArea,
-            "bottom": Qt.BottomDockWidgetArea
-        }.get(info.pos, Qt.LeftDockWidgetArea)
-        if info.show:
-            areaDockWidgets = [
-                dockWidget_ for dockWidget_ in self.mainWindow.findChildren(QDockWidget)
-                if self.mainWindow.dockWidgetArea(dockWidget_) == area
-            ]
-            if areaDockWidgets:
-                self.mainWindow.tabifyDockWidget(areaDockWidgets[-1], dockWidget)
-            else:
-                self.mainWindow.addDockWidget(area, dockWidget)
-        self._dockWidgets[name].append(dockWidget)
+        if info.pos == "center":
+            wrapperWidget = QMdiSubWindow(self.centralWidget)
+            wrapperWidget.setWindowTitle(name)
+            wrapperWidget.setWidget(frame)
+            wrapperWidget.show()
+        else:
+            wrapperWidget = QDockWidget(name, self.mainWindow)
+            wrapperWidget.setWidget(frame)
+            area = {
+                "left": Qt.LeftDockWidgetArea,
+                "right": Qt.RightDockWidgetArea,
+                "top": Qt.TopDockWidgetArea,
+                "bottom": Qt.BottomDockWidgetArea
+            }.get(info.pos, Qt.LeftDockWidgetArea)
+            if info.show:
+                areaDockWidgets = [
+                    dockWidget_ for dockWidget_ in self.mainWindow.findChildren(QDockWidget)
+                    if self.mainWindow.dockWidgetArea(dockWidget_) == area

```

Comment:
Oops, I'm sorry for the late review. Now there is not `dockWidget`, so you don't have to put the trailing underscore `dockWidget_`.